home/dot_config/mackup/.mackup/alrair.cfg

```diff
@@ -0,0 +1,10 @@
+[application]
+name = Altair GraphQL Client
+
```

Comment:
This file is mispelled -- please do a double check on all the files.

Also, these files should not be templated unless absolutely necessary. Check out this file https://github.com/lra/mackup/blob/master/mackup/applications/vscode.cfg and skim through the mackup documentation to see how we can backup the /snap/ data as well.

Reply:
This is fixed. I remember fixing the same typo, may be that was in the `mackup.cfg` file.

I have removed the templating from these cfg files. I did it to keep it relevant only to the OS, and to avoid potential errors due to folders/files not being found but that is handled by the application.